README.md

```diff
@@ -88,22 +88,25 @@ fabrica [down | start | stop]
 Downs, starts or stops the Hyperledger Fabric network for configuration in the current directory.
 This is similar to down, start and stop commands for Docker Compose.
 
-### recreate
+### prune
 
 ```bash
-fabrica.sh recreate [/path/to/fabrica-config.json]
+fabrica prune
 ```
 
-Downs the network, removes `fabrica-target` directory and starts the Hyperledger Fabric network for given Fabrica configuration file.  
-Useful when you edited `fabrica-config` file and want to start newer network version in one command.    
+Downs the network and removes `fabrica-target` directory.
 
-### prune
+### restart, reboot, recreate
 
 ```bash
-fabrica prune
+fabrica.sh restart
+fabrica.sh reboot
+fabrica.sh recreate [/path/to/fabrica-config.json]
 ```
 
-Downs the network and removes `fabrica-target` directory.
+* `restart` -- just stops and starts the network. Useful in case of minor config changes of generated files.
```

Comment:
`reset` is redundant, as all configtx changes are applied at the start of network.  
I would remove it for now and leave only reboot.

Reply:
✅  removed